.github/workflows/cmake.yml

```diff
@@ -18,22 +18,16 @@ jobs:
           - name: Ubuntu 20.04 GCC
             os: ubuntu-20.04
             compiler: gcc
+            cmake-args: -DWITH_BENCHMARKS=ON
             cxx-compiler: g++
 
           - name: Ubuntu GCC ASAN
             os: ubuntu-latest
             compiler: gcc
             cxx-compiler: g++
-            cmake-args: -DWITH_SANITIZER=Address
+            cmake-args: -DWITH_SANITIZER=Address -DWITH_BENCHMARKS=ON
```

Comment:
Seems to need `-Wno-maybe-uninitialized` to build without warnings/errors.

Reply:
Added 👍